# generated from tests.py by /home/cs1521/public_html/19T3/scripts//autotest_update_from_solution

max_cpu=30
command=1521 spim -f print_bigger.s
files=print_bigger.s


0 stdin="1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n" description="1 2 3 4 5 6 7 8 9 10" expected_stdout="Loaded: /home/cs1521/share/spim/exceptions.s\n10\n"
1 stdin="1\n2\n3\n4\n5\n6\n7\n8\n9\n5\n" description="1 2 3 4 5 6 7 8 9 5" expected_stdout="Loaded: /home/cs1521/share/spim/exceptions.s\n5\n6\n7\n8\n9\n5\n"
2 stdin="1\n2\n3\n4\n5\n6\n7\n8\n9\n1\n" description="1 2 3 4 5 6 7 8 9 1" expected_stdout="Loaded: /home/cs1521/share/spim/exceptions.s\n1\n2\n3\n4\n5\n6\n7\n8\n9\n1\n"
3 stdin="1\n2\n3\n4\n5\n6\n7\n8\n9\n0\n" description="1 2 3 4 5 6 7 8 9 0" expected_stdout="Loaded: /home/cs1521/share/spim/exceptions.s\n1\n2\n3\n4\n5\n6\n7\n8\n9\n0\n"
4 stdin="5\n5\n5\n5\n5\n4\n5\n5\n5\n5\n" description="5 5 5 5 5 4 5 5 5 5" expected_stdout="Loaded: /home/cs1521/share/spim/exceptions.s\n5\n5\n5\n5\n5\n5\n5\n5\n5\n"
5 stdin="5\n5\n5\n5\n5\n4\n5\n5\n5\n4\n" description="5 5 5 5 5 4 5 5 5 4" expected_stdout="Loaded: /home/cs1521/share/spim/exceptions.s\n5\n5\n5\n5\n5\n4\n5\n5\n5\n4\n"
6 stdin="25664\n11593\n12172\n17\n30727\n8527\n23102\n26011\n28344\n5565\n" description="25664 11593 12172 17 30727 8527 23102 26011 28344 5565" expected_stdout="Loaded: /home/cs1521/share/spim/exceptions.s\n25664\n11593\n12172\n30727\n8527\n23102\n26011\n28344\n5565\n"
